universal-application-tool-0.0.1/app/services/program/ProgramService.java

```diff
@@ -42,6 +42,8 @@
    */
   ProgramDefinition getProgramDefinition(long id) throws ProgramNotFoundException;
 
+  ActiveAndDraftPrograms getActiveAndDraftPrograms();
```

Comment:
Suggestion (blocking): please document this method

Reply:
Done.